substratevm/mx.substratevm/suite.py

```diff
@@ -570,46 +599,6 @@
             "spotbugs": "false",
         },
 
-        "com.oracle.svm.jfr": {
```

Comment:
I think it would be better to have the JFR implementation (and its dependencies) in a separate project. Pretty much all VM-internal events should use the `JfrNativeEventWriter` anyways (via some JDK-independent abstraction).